servers/src/test/java/tachyon/worker/block/meta/StorageDirTest.java

```diff
@@ -362,6 +360,32 @@ public void resizeTempBlockMetaTest() throws Exception {
     Assert.assertEquals(TEST_DIR_CAPACITY - newSize, mDir.getAvailableBytes());
   }
 
+  @Test
+  public void resizeTempBlockMetaInvalidStateExceptionTest() throws Exception {
+    mDir.addTempBlockMeta(mTempBlockMeta);
+    final long newSize = TEST_TEMP_BLOCK_SIZE - 10;
+    try {
```

Comment:
Q: Why use traditional `try-catch` structure instead of more convenient `junit @Rule ExpectedException` component here?
A: For this PR, I changed the time when `resizeTempBlockMeta` should work. The correctness of tempBlockSize should be checked after an expected Exception thrown. `Junit` just cannot do additional check when Exception thrown.